exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function first
-
+function sum(a,b) {
+    return a + b;
+  }
 // Call the function and assign to a variable `sum`
-
+sum = sum(5,6);
 console.log(sum);
```

Comment:
After line 6, you won't be able to use the sum function to add numbers, any ideas why this might be the case?

Reply:
sum is now always going to equal sum(5,6), i should have used a new variable here